theme/components/linkballoonpanel.scss

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2003-2016, CKSource - Frederico Knabben. All rights reserved.
+// For licensing, see LICENSE.md or http://ckeditor.com/license
+
+.ck-link-balloon-panel {
+	padding: ck-spacing();
+
+	// Box to accommodate buttons.
+	.ck-link-form-actions {
```

Comment:
I think it could go outside `.ck-link-balloon-panel`. I would shorten the selectors by 1 level. I think it's quite unlikely that the `.ck-link-form__actions` will appear outside `.ck-link-balloon-panel` in the nearest future.

Both could then be wrapped like

```
.ck-link {
  &-balloon-panel {}
  &-form__actions {}
}
```